src/screens/crew6/engineeringScreen.cpp

```diff
@@ -476,3 +475,8 @@ void EngineeringScreen::addSystemEffect(string key, string value)
     }
     system_effects_index++;
 }
+
+string EngineeringScreen::toNearbyInt(float value)
```

Comment:
The solution is fine, but the function name is misleading, as you would assume it returns an int from the name.

Reply:
I felt the same way about `nearbyint`, which returns a float.

Reply:
Clarified the function name.